lib/loggerator_rails.rb

```diff
@@ -0,0 +1,77 @@
+require "loggerator"
+require "loggerator_rails/log_subscriber"
+
+# Using LoggeratorRails namespace over Loggerator::Rails due to the fact that
+# when Loggerator is included in Rails classes, e.g. ActiveRecord::Base,
+# Loggerator::Rails takes precedence over the Rails namespace, causing things
+# like "Rails.env" to break without prefixing "::". This is not only annoying,
+# but could break existing projects which upgrade to this version of Loggerator.
+module LoggeratorRails
+  # Implementation respectfully borrowed from:
+  # https://github.com/minefold/scrolls-rails
+  extend self
+
+  def setup(_app)
+    detach_existing_subscribers
+    LoggeratorRails::LogSubscriber.attach_to(:action_controller)
+  end
+
+  def subscribe?
+    return true unless defined?(@@subscribe) # ensure default is true
+
+    @@subscribe
```

Comment:
That doesn't seem to work as expected... 

```
[3] pry(main)> LoggeratorRails.use_default_subscribers!
=> false
[4] pry(main)> LoggeratorRails.class_variable_get("@@subscribe")
=> false
[5] pry(main)> LoggeratorRails.subscribe?
=> true
```

When I saw your comment, I 🤦‍♂️ , but then upon trying it, I got the above.

Tried both

```ruby
@@subscribe ||= true
```

... and ...

```ruby
@@subscribe ||= true
@@subscribe
```

... with the same result.